tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1120,8 +1149,10 @@ public:
 
 template <typename ResultType>
 class R__CLING_PTRCHECK(off) SumHelper : public RActionImpl<SumHelper<ResultType>> {
-   const std::shared_ptr<ResultType> fResultSum;
+   std::shared_ptr<ResultType> fResultSum;
```

Comment:
why did you remove the const-ness?

Reply:
Because there was no const-ness in the Kahan sum tutorial where the Kahan class was defined. I think I can apply the const again, if that's better?

Reply:
Here it's not the number that is marked const, it's the pointer -- this is a constant shared pointer to a non-constant number.

Note that in general const data members are discouraged: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-constref -- but in this case it's also a valid approach to leave the code as you found it :)